syncstorage/storage/spanner.py

```diff
@@ -53,6 +53,9 @@
 
 EPOCH = datetime.datetime.utcfromtimestamp(0)
 
+# a bogus FIND_ITEMS bind param
+BOGUS_ID = object()
```

Comment:
Is this used (or planned to be used) anywhere outside of the SpannerStorage.__init__()?

Reply:
it's only used in _find_items as basically a placeholder